README.md

```diff
@@ -7,47 +8,33 @@ Esse projeto é uma iniciativa do CocoaHeads Brasil que tem como objetivo de tra
 
 Link do livro original: [Swift.org](https://swift.org/documentation/TheSwiftProgrammingLanguage(Swift2.2).epub)
 
-TODO: Melhorar essa descrição e corrigir o link
-TODO: Adicionar screenshot com preview do livro no GitBook
-
 ---
 
 ###Como contribuir?
 
-Basta criar os documentos traduzidos em arquivos *.md* específico, seguindo a estrutura sugerida pelo SUMMARY.md (abrir raw do arquivo para mais detalhes). A formatação do SUMMARY.md é importante, pois ela que gera o
-
-### Como utilizar o GitBook?:
-
-Para a tradução desse livro, estamos utilizando o [GitBook](http://gitbook.com).
-
-O GitBook pode ser instalado utilizando o **NPM**, execute:
-
-```
-$ npm install gitbook-cli -g
-```
+Passos para começar a ajudar:
 
-Para criar um arquivo SUMMARY.md com a estrutura básica para o livro, execute:
+1. Os capítulos disponíveis para tradução são criado em forma de issue no Github, por exemplo, Issue 3: Generics;
+2. O arquivo *.md* é criado com o nome do capítulo, por exemplo, Generics;
+3. Caso queira contribuir com determinado capítulo, comente na issue para não haver traduções múltiplas do mesmo arquivo;
 
-```
-$ gitbook init
-```
+** Crie uma branch para determinado arquivo que queira traduzir e faça o Pull Request.
 
-Você pode iniciar um server local do GitBook (disponível num http://localhost:4000/ ou algo similar), execute:
+Caso queira observar como está ficando o texto, basta utilizar o Gitbook e seguir as instruções.
+(https://github.com/CocoaHeadsBrasil/the-swift-programming-language-in-portuguese-br/wiki) 
```

Comment:
Acho que aqui pode ser assim:

Caso queira observar como está ficando o texto, basta utilizar o Gitbook e seguir as instruções na [página do wiki](https://github.com/CocoaHeadsBrasil/the-swift-programming-language-in-portuguese-br/wiki)